webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -671,7 +681,12 @@ Kommunicate.richMsgEventHandler = {
             (messagePxy['metadata'] = msgMetadata);
         (Object.keys(msgMetadata).length > 0 ||
             Object.keys(messagePxy).length > 0) &&
-            Kommunicate.sendMessage(messagePxy);
+        Kommunicate.sendMessage(messagePxy);
+        postFormDataAsMessage && 
+        Kommunicate.sendMessage({
```

Comment:
There is a send message above it, are we sending 2 messages for one form submission?